frontend/pages/index.tsx

```diff
@@ -8,7 +8,6 @@ import { signIn, getSession, signOut } from "next-auth/react";
 
 /* Internal Imports */
 import Layout from "components/Layout";
-import InfoTooltip from "components/InfoTooltip";
```

Comment:
InfoTooltip is used so we can't remove it, check 
https://github.com/tonykogias/optimism-faucet/blob/44f5ffb1a283f01d666ab7c1459f584200b272b9/frontend/pages/index.tsx#L82

Reply:
@aragalie After fixing this, I think it's good to go! Please try to squash your commits into 1 for such a small change.